### PR TITLE
Create grafica.app.xml

### PR DESCRIPTION
**Pull request by jumde**
Adding ruleset for grafica.app

### CODE REVIEW

File: src/chrome/content/rules/grafica.app.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!--
+grafica.app/ 
+-->
```
Comment (zoracon) [inline src/chrome/content/rules/grafica.app.xml]: Could you please clarify this comment?

Comment (jumde) [reply src/chrome/content/rules/grafica.app.xml]: @zoracon I added this to highlight that the rule set will apply to all `grafica.app` urls, I can remove it if it feels unnecessary.

Comment (pipboy96) [reply src/chrome/content/rules/grafica.app.xml]: @jumde Yes, it's not really needed here.

Comment (jumde) [reply src/chrome/content/rules/grafica.app.xml]: @pipboy96 @zoracon - Updated.

File: src/chrome/content/rules/grafica.app.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<ruleset name="grafica.app">
```
Comment (pipboy96) [inline src/chrome/content/rules/grafica.app.xml]: Add before this line:

```
<!--
	NXDOMAIN: www
-->
```

Comment (jumde) [reply src/chrome/content/rules/grafica.app.xml]: @pipboy96 - Added

**Comment by jumde** [comment]
Thanks for the approval @pipboy96

**Pull request merged**